### PR TITLE
Added instructions explaining how to move certificate from user to root store

### PR DESCRIPTION
**Pull request by DemanNL**
Added instructions explaining how to move the burp certificate from user to root store.

You can also add alternative certificate installation instructions. The current instructions download the cert outside android VM, push it and install it. This can all be done from within the Android VM as well.

The instructions would be as followed:

1. Start Burp and configure your Android VM Wi-Fi settings to use Burp proxy.
2. Visit http://burp, and download the certificate.
3. Rename the certificate to cacert.cer.
4. Open the file and you will be prompted to confirm installation of the certificate.

Let me know what you think about the alternative instructions as well and if it would fit in the document or not.

### CODE REVIEW

File: Document/0x05b-Basic-Security_Testing.md [hunk 1]
```diff
@@ -756,6 +756,12 @@ An easy way to install a CA certificate is to push the certificate to the device
 
 You should then be prompted to confirm installation of the certificate (you'll also be asked to set a device PIN if you haven't already).
 
+This installs the certificate in the user certificate store. In order to place the certificate in the root store you can perform the following steps:
+1. Run adb as root with `adb root` and `adb shell`.
+2. Locate the newly installed certificate at `/data/misc/user/0/cacerts-added/`.
+3. Copy the certificate to the following folder `/system/etc/security/cacerts/`.
+4. Reboot the Android VM.
```
Comment (sushi2k) [inline Document/0x05b-Basic-Security_Testing.md:763]: This is unfortunately already outdated and will only work for Android versions below 7.0. We should put in a sentence instead saying that Network Security Configuration is enforcing that user installed CAs are not trusted by default anymore. 
There is the manual option of getting the Burp CA into the System truststore (Bypassing the Network Security Configuration) and the one we described in the MSTG in "Bypassing the Network Security Configuration"

Do you want to re-structure the paragraph instead?

**Review by sushi2k — CHANGES_REQUESTED** [review]
See comment

**Comment by sushi2k** [comment]
Just updated my comment, apologies was referring to hardware devices originally not to VMs; I changed my comment. Please have a look! 
Thank you @DemanNL

**Comment by sushi2k** [comment]
Got update from proxy77 in Slack: "it works for me in android version 10 (geneymotion custom device)"

**Comment by cpholguera** [comment]
I was going thought the PRs, here only this comment https://github.com/OWASP/owasp-mstg/pull/1915#discussion_r667701885 is open, right? @DemanNL please let us know when you address that to make a final review. Thanks in advance!

**Review by DemanNL — COMMENTED** [review]
Added on which VM this was tested. However I'm sure this works on any rooted machine.

**Comment by DemanNL** [comment]
> 
> 
> I was going thought the PRs, here only this comment [#1915 (comment)](https://github.com/OWASP/owasp-mstg/pull/1915#discussion_r667701885) is open, right? @DemanNL please let us know when you address that to make a final review. Thanks in advance!

@cpholguera is everything okay now?

**Review by TheDauntless — APPROVED** [review]

**Review by sushi2k — APPROVED** [review]

**Pull request merged**